DevoxxClientMobile/src/main/java/com/devoxx/views/helper/Util.java

```diff
@@ -199,4 +207,38 @@ public static void showToast(String message, Duration duration)  {
     public static String safeStr(String s) {
         return s == null? "": s.trim();
     }
+
+    public static void hidePastConferenceMessage() {
+        if (pastConferenceMessage != null) {
+            pastConferenceMessage.hide();
+        }
+    }
+
+    public static void showPastConferenceMessage() {
+        if (pastConferenceMessage == null) {
```

Comment:
The label to show the message shows up too soon, before the catalog is loaded. Maybe it should be showed afterwards?

Reply:
Yes, persistence is by design. The label is more useful when user re-opens the app and doesn't remember their last selection.

Is there a way to reproduce when the label appears over the conference selector?

Reply:
I just saw it once, testing on desktop. Never mind for now.